doc/cli.markdown

```diff
@@ -959,7 +960,7 @@ support) please check:
 
 #### --port, -p &#60;port&#62;
 
-SSH gateway port
+SSH gateway port (by default balenaOS listens on `22222`)
```

Comment:
hmm "gateway port" is ***not*** the port that a ssh server listens on a device, right? I think it refers to the port that some cloud service or openBalena service listens on, maybe a VPN service (?), but in any case "balenaOS" would not be relevant here, because the whole thing goes over the cloud/VPN. Am I misunderstanding it? I could well be wrong, but I think that "balena ssh" is fundamentally different to plain ssh as in:
```
ssh -p 22222 root@192.168.2.5
```

Reply:
I was under the impression that ssh'ing to a device in local mode would initiate a direct ssh connection (rather than going over the VPN), which is what I intended to document here.

Reply:
It does, the assertion that it goes over the VPN is incorrect. balenaOS is listening for SSH traffic on port 22222, regardless of development or production edition or local mode activation. In production mode, the port is not open and a specific device key is required (and in fact users can register these keys using the `sshKeys` property in `config.json` on-device. I agree with the modified text.

Reply:
So, there's some misinformation here:

`balena ssh <uuid>` will always use the proxy/VPN
`balena ssh <x.local>` and `balena ssh x.x.x.x` will use a direct SSH connection to port `22222` on the local device

Because of this I think it's probably *really important* to make that distinction in the documentation. But I think @xginn8 is right in documenting that balenaOS is listening on port 22222, but it's not really a gateway port as it's not doing any forwarding. It's just the device's local SSH port.

Reply:
> `balena ssh x.x.x.x` will use a direct SSH connection

So it does! I stand corrected. Sorry for the noise @xginn8, @hedss. Documenting it as Hedley suggested would be nice.

Reply:
I have updated the documentation accordingly, thanks for the feedback both! I'm glad to clear this up